cypress/integration/first_tests.specs.js

```diff
@@ -0,0 +1,11 @@
+const HOST = `http://localhost:1234` // Please adjust to your own app port
+
+describe(`My first test`, () => {
+  before(() => {
+    cy.visit(HOST) // Visit our app
+  })
+
+  it('Should visit the dashboard', () => {
+    cy.url().should('match', new RegExp(HOST)) // The current host URL
+  })
+})
```

Comment:
Could you add tests that guarantee that all UI elements render correctly? As this is a getting started test

Reply:
Added tests for checking if searchbars is visible and autocomplete too (during typing)